src/Donations/Micros/GiftForm.tsx

```diff
@@ -33,14 +33,15 @@ export default function GiftForm({}: Props): ReactElement {
   }, [isGift]);
 
   const onSubmit = (data: any) => {
-    setgiftDetails({ ...giftDetails, ...data });
+    setgiftDetails({ ...giftDetails, ...data, type: "invitation" });
   };
 
   const resetGiftForm = () => {
     const defaultDeails = {
```

Comment:
@Shreyaschorge
1. Correct `defaultDeails` to `defaultDetails` within this file.
2. Should `defaultDetails` defined on line 16-20 also contain `type`?

Reply:
@mohitb35 

There is no need to define type on line 16-20 since that defaultDetails sets default value to the fields which will be registered with react-hook-form.

This useEffect will take care of introducing default type to defaultDetails
https://github.com/Plant-for-the-Planet-org/planet-donations/blob/ca9b3d086c88e0e2868512cd71b5a67ad17ed488/src/Donations/Micros/GiftForm.tsx#L27